src/index.html

```diff
@@ -0,0 +1,13 @@
+<!doctype html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport"
+          content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>Document</title>
+  </head>
+  <body>
+  Hello, world!
+  </body>
```

Comment:
```suggestion
  <body>
    Hello, world!
  </body>
```
Pay attention to the formatting. There should be some indentations
You may read about it here https://mate-academy.github.io/style-guides/htmlcss.html